app/src/main/java/model/BaseFragment.kt

```diff
@@ -0,0 +1,38 @@
+package model
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+
+abstract class BaseFragment() : Fragment() {
+    lateinit var articleList: Array<Article>
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        initArticleList()
+    }
+//    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+//        super.onViewCreated(view, savedInstanceState)
+//        initArticleList()
+//    }
```

Comment:
do you need the comment above?

Reply:
I was debating which method to override.
The task required me to override the onViewCreated method, but in this way the app crashes ...
And when I override the onCreate method there was no problem.